src/apack/zcl_abapgit_apack_reader.clas.abap

```diff
@@ -98,6 +102,17 @@ CLASS zcl_abapgit_apack_reader IMPLEMENTATION.
     lv_descriptor_cust = zif_abapgit_apack_definitions=>c_apack_interface_cust && '~DESCRIPTOR'.
     lv_descriptor_sap  = zif_abapgit_apack_definitions=>c_apack_interface_sap && '~DESCRIPTOR'.
 
+    lv_class_name = cl_abap_classdescr=>get_class_name( io_manifest_provider ).
+    SPLIT lv_class_name AT '\CLASS=' INTO lv_empty lv_class_name.
+
+    IF lv_class_name(1) = '/'.
+      CREATE OBJECT lo_nspc.
```

Comment:
let's replace this with `zcl_abapgit_factory=>get_sap_namespace( )->split_by_name( ... )`

Reply:
Sounds good.